apps/client/src/features/app-settings/panel/automations-panel/template-input/templateInput.utils.ts

```diff
@@ -18,15 +18,16 @@ const staticAutocompleteOptions = [
   '{{timer.expectedFinish}}',
   '{{timer.secondaryTimer}}',
   '{{timer.startedAt}}',
-  '{{runtime.selectedEventIndex}}',
-  '{{runtime.numEvents}}',
-  '{{runtime.offset}}',
-  '{{runtime.plannedStart}}',
-  '{{runtime.plannedEnd}}',
-  '{{runtime.actualStart}}',
-  '{{runtime.expectedEnd}}',
-  '{{currentGroup.id}}',
-  '{{currentGroup.startedAt}}',
+  '{{rundown.selectedEventIndex}}',
```

Comment:
Question: do we need to do anything in the server to support this?

Reply:
dont think so, will check